mobile/src/main/java/org/openhab/habdroid/util/Constants.java

```diff
@@ -40,4 +40,18 @@
             public static final int ALWAYS = 5;
         }
     }
-}
\ No newline at end of file
+    public static final String PREFERENCE_CUSTOM_BROADCAST    = "default_openhab_custom_broadcast_enabled";
+    public static final String PREFERENCE_CUSTOM_BROADCAST_BROADCAST = "default_openhab_custom_broadcast";
+    public static final String PREFERENCE_CUSTOM_BROADCAST_EXTRA = "default_openhab_custom_broadcast_extra";
+    public static final String PREFERENCE_CUSTOM_BROADCAST_ITEM = "default_openhab_custom_broadcast_item";
+
+    public interface ACTION {
+        public static String MAIN_ACTION = "org.openhab.habdroid.action.main";
```

Comment:
This doesn't seem to be actually used. It's set in the notification pending intent, but is never checked against.

Reply:
Removed that